credential-exchange-types/src/format.rs

```diff
@@ -156,36 +156,16 @@ pub enum Extension<E = ()> {
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(tag = "type", rename_all = "kebab-case")]
+#[allow(clippy::large_enum_variant)]
```

Comment:
We should figure out if we want to box everything. Slightly less ergonomic but I guess we save some memory.

Reply:
Box isn't all that un-ergonomic. You can `.into()` most of the time to box the type. And with it implementing `Deref` and `DerefMut` you can destructure and pattern match all you want